tests/instances.tcl

```diff
@@ -15,6 +15,7 @@ source ../support/util.tcl
 source ../support/aofmanifest.tcl
 source ../support/server.tcl
 source ../support/test.tcl
+source ../support/response_transformers.tcl
```

Comment:
@tezc please rebase, both lines 18 and 41 should be deleted

Reply:
done